frontend/packages/kubevirt-plugin/src/constants/vm/provision-source.ts

```diff
@@ -63,7 +58,25 @@ export class ProvisionSource extends SelectDropdownObjectEnum<string> {
     {},
   );
 
-  static getAll = () => ProvisionSource.ALL;
+  static getVMSources = () => [
```

Comment:
the naming might be a bit misleading since this is used also for the templates in the create-vm-wizard

Reply:
hm, right. I changed it to getAdvancedWizardSources/getBasicWizardSources, not sure if thats better.

Reply:
yes, looks good now